core/src/main/java/hudson/model/View.java

```diff
@@ -334,6 +335,14 @@ public void setDescription(String description) {
         return r;
     }
 
+    /**
+     * @return all the {@link ViewPropertyDescriptor}s that can be potentially configured on this View and are visible
+     * for the user. Use {@link DescriptorVisibilityFilter} to make a View property invisible for users.
+     */
```

Comment:
If it is not an override ...

```suggestion
     * @since TODO
     */
```

Reply:
done.